app/UserPreferences.m

```diff
@@ -148,9 +148,11 @@ - (BOOL)isEqual:(id)object {
 
 NSDictionary<NSString *, Theme *> *presetThemes;
 + (void)initialize {
-    presetThemes = @{@"Light": [self _themeWithForegroundColor:UIColor.blackColor
+    presetThemes = @{@"Light":         [self _themeWithForegroundColor:UIColor.blackColor
```

Comment:
but the :s aren’t aligned anymore

Reply:
(in the method call I mean)

Reply:
I aligned the [ instead:
![image](https://user-images.githubusercontent.com/8685248/51074722-0021ef80-168b-11e9-9e3e-dfe795ff9d56.png)
anyways in the current file the [ are aligned

Reply:
weren’t they both aligned initially?